extensions/wikia/Optimizely/Optimizely.class.php

```diff
@@ -6,6 +6,14 @@
  *
  */
 class Optimizely {
+
+	/**
+	 * Which environments are considered a developemnt/testing environments for Optimizely experiments?
+	 * Optimizely Wikia Dev project script should be served also on sandboxes, to prevent poluting Wikia Prod project
+	 * script with experiments that are still under developement.
+	 */
+	const OPTIMIZELY_DEV_ENVIRONMENTS = [ WIKIA_ENV_DEV, WIKIA_ENV_SANDBOX ];
```

Comment:
I didn't see that recent change in Mercury. :open_mouth:  Then it would be consistent to serve "Wikia [production]" Optimizely project on sandboxes on Oasis, too. I'll just cut the PR to just serve the code on devbox and sandbox from original CDN. Thanks for the insights! :bow: